src/Common/tests/System/Net/Capability.Security.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+
+namespace System.Net.Test.Common
+{
+    public static partial class Capability
+    {
+        private const string CARootThumbprint = "3B279AD43D6DD459268D3F3A3D72DAAD4BF4D9C6";
+        private static Lazy<bool> trustedCertificateSupport = 
+            new Lazy<bool>(IsTrustedCertificateSupported, LazyThreadSafetyMode.ExecutionAndPublication);
+        
+        public static bool TrustedCertificate()
```

Comment:
I'm not sure `TrustedCertificate` is the best word choice. First, it's really checking for a Trusted Root Certificate and not an end-entity server certificate that happens to be trusted. Second, I think this method need a verb.

Maybe `IsTrustedRootInstalled` would be a better name?